app/models/validations/setup_validations.rb

```diff
@@ -1,5 +1,14 @@
 module Validations::SetupValidations
   include Validations::SharedValidations
+  include CollectionTimeHelper
+
+  def validate_startdate_setup(record)
+    return unless record.startdate && date_valid?("startdate", record)
+
+    unless record.startdate.between?(active_collection_start_date, current_collection_end_date) || !FeatureToggle.startdate_collection_window_validation_enabled?
```

Comment:
minor thing, can:
```
!FeatureToggle.startdate_collection_window_validation_enabled?
```
be made into a short circuit at the top of the method eg:
```
return if !FeatureToggle.startdate_collection_window_validation_enabled?
```
if so, it should read more easily?

Reply:
good idea, will do this for sales too 👍